src/app/common/components/Auth/PasswordStrengthIndicator/PasswordStrengthIndicator.tsx

```
@@ -0,0 +1,55 @@
+        setRequirements(newRequirements);
+        setIsPasswordValid(fulfilledRequirements === TOTAL_REQUIREMENTS);
+    }, [fulfilledRequirements, password, setIsPasswordValid]);
+
+    return (
+        <div className="password-strength-bar">
+            {Array.from({ length: TOTAL_REQUIREMENTS }).map((_, index) => {
+                const isFulfilled = index < fulfilledRequirements;
+                return (
+                    <div
+                        className={`segment ${isFulfilled ? 'green' : 'gray'}`}
+                    />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)